src/components/Camera/style.scss

```diff
@@ -102,8 +102,9 @@
 .webcamContainer {
   position: absolute;
   top: 0;
+  left: 0;
```

Comment:
I think you need to change this on other media queries as well

Reply:
Good point. The layout displays fine on the mobile devices I tested as is so didn't think to change it, but I'll check if they are required to be sure.